packages/react-strict-dom/tests/__snapshots__/css-test.native.js.snap-native

```diff
@@ -1011,6 +1008,7 @@ exports[`properties: logical direction inset: inset vs top 1`] = `
 {
   "style": {
     "bottom": 100,
+    "insetBlockStart": 3,
```

Comment:
This looks like a regression

Reply:
This is how RN works, so this might be expected.